### PR TITLE
Do not convert include_granted_scopes to bool

### LINKED ISSUE #2454: Invalid boolean conversion for extraAuthorizeParameters include_granted_scopes
When trying to define an OpenIDConnect ID provider in Ansible hosts by setting include_granted_scopes. If the value is “true” with the double quotes then the definition is invalid. 

For instance extraAuthorizeParameters': {'include_granted_scopes': ‘\”true\"'} results in include_granted_scopes: false in the configuration.

The issue seems to be that the value of include_granted_scopes is evaluated as ansible_bool.

https://github.com/openshift/openshift-ansible/blob/b8e72ea2143834317043f19cbc3dddf8ad9047a5/filter_plugins/openshift_master.py#L388

### PR DESCRIPTION
**Pull request by smunilla**
Fixes #2454

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: filter_plugins/openshift_master.py [hunk 1]
```diff
@@ -386,7 +386,7 @@ def __init__(self, api_version, idp):
 
         if 'extraAuthorizeParameters' in self._idp:
             if 'include_granted_scopes' in self._idp['extraAuthorizeParameters']:
-                val = ansible_bool(self._idp['extraAuthorizeParameters'].pop('include_granted_scopes'))
+                val = self._idp['extraAuthorizeParameters'].pop('include_granted_scopes')
```
Comment (detiber) [inline filter_plugins/openshift_master.py]: just realize that I had forgotten to actually "submit" my review...  uggh

Comment (sdodson) [inline filter_plugins/openshift_master.py]: @smunilla bump

**Comment by smunilla** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
Closing this PR as it is stale and needs to be rebased.   Please reopen the PR if you feel this patch is still applicable.  Thanks!

**Comment by enj** [comment]
This needs to be fixed for https://bugzilla.redhat.com/show_bug.cgi?id=1488505

@sdodson @mtnbikenc @smunilla

**Pull request closed without merging**